scripts/fiber/tests-failing.txt

```diff
@@ -48,6 +48,7 @@ src/renderers/shared/__tests__/ReactPerf-test.js
 
 src/renderers/shared/hooks/__tests__/ReactComponentTreeHook-test.js
 * can be retrieved by ID
+* works
```

Comment:
I use `Set` in DEV mode earlier so the test that specifically removed `global.Set` now fails.
We use `Set` in other places anyway so I think this doesn’t really matter.
We should fix those together (or depend on a polyfill).